.travis.yml

```diff
@@ -31,4 +31,4 @@ jobs:
         - flake8 monitoring_service
 
       script:
-        - coverage run --source monitoring_service/ --omit 'monitoring_service/test/*' -m py.test --travis-fold=always -vvvvvvvvs $TEST_TYPE
```

Comment:
Do you still want to keep the `-x` after the discussion in RC?

Reply:
In order to have a shorter build push-CI breaks loop I'll leave it here for now. It can be removed once the code stabilizes.